contracts/solidity/test/Transfer.sol

```diff
@@ -11,7 +11,8 @@ contract TransferTest is Test {
         address snMessagingAddress = 0xde29d060D45901Fb19ED6C6e959EB22d8626708e;
         uint256 snEscrowAddress = 0x0;
         uint256 snEscrowWithdrawSelector = 0x15511cc3694f64379908437d6d64458dc76d02482052bfb8a5b33a72c054c77;
```

Comment:
are this values better if they are read from .env ?

Reply:
idk if this is necessary because it's the test file